rt_addon/strings.xml

```diff
@@ -0,0 +1 @@
+<label>The following will be localized from an addons strings - $ADDON[plugin.video.russia-today 32001]</label>
```

Comment:
Where is this used? Strings.xml support was dropped in matrix althought this location is wrong even in leia.
Localisation files are stored in resources/language/resource.language.yourlanguage/strings.po (gettext format). See https://kodi.wiki/view/Language_support